### DISCUSSION TITLE
How can I disable the tooltip when hovering over a RichText label?

### DISCUSSION THREAD

**Discussion by kazukazu123123** [emilk/egui#5294] [Q&A]
Currently, I'm using this code:
```rust
ui.label(RichText::new("test (insert long text here...)").color(Color32::RED));
```
However, when I hover over the text, a tooltip appears. I would like to disable this tooltip, but I'm not sure how to do it. Could someone please explain how to achieve this?

![image](https://github.com/user-attachments/assets/24775866-7aac-4816-ac93-0d556688bfff)

**Comment by lucasmerlin** [comment]
Seems like its hardcoded to show the hover text when it was truncated: 
https://github.com/emilk/egui/blob/707cd03357f926d040acb241a5c42d8e91a83d8e/crates/egui/src/widgets/label.rs#L256-L259

Seems like someone already opened an issue for this, so I think we can close this discussion: https://github.com/emilk/egui/issues/4427

**Reply by kazukazu123123** [reply]
Thanks for letting me know.
